README.md

```diff
@@ -7,8 +7,12 @@ This project uses go modules, ensure that go version 1.12 or above is in use.
 If on go 1.12 then ensure the project either resides outside of your `GOPATH` or `GO111MODULE` is set to true
 
 1. To start the service use make, `make debug`
+2. Clone `sixteens`
```

Comment:
This should be adjusted as some people would already have sixteens setup locally.

I would rather just go with:
- Run sixteens locally using `npm run dev`
-Run `make debug` to start the service on http://localhost:24400